platform/ios/detect.py

```diff
@@ -32,6 +32,7 @@ def get_opts():
             "",
         ),
         ("IOS_SDK_PATH", "Path to the iOS SDK", ""),
+        ("apple_target_triple", "Triple for corresponding target Apple platform toolchain", ""),
```

Comment:
Better if add back `ios_triple` for compability
```suggestion
        ("ios_triple", "Deprecated, use 'apple_target_triple' instead", ""),
        ("apple_target_triple", "Triple for corresponding target Apple platform toolchain", ""),
```

Reply:
For this I'm going to disagree, if it works at all, I think it's such a rare use case that it's going to be bespoke and easy to update to the new value on the spot. Let me know if you disagree.

Reply:
This is a [documented](https://docs.godotengine.org/en/stable/contributing/development/compiling/cross-compiling_for_ios_on_linux.html) feature, and the latest reply of the document happens at the end of last year. That means some user is still using cross compiling for their needs.

From the documentation, libtool for cross compiling on Linux is using third party [cctools-port](https://github.com/tpoechtrager/cctools-port). Looking at the example of the latest cctools-port, target triple is still used. The option `ios_triple` exists for reason, not as rarely used as it seems.

Since the intent of this PR is to add visionOS support, I think is better to not break these iOS features. After renaming the build option, it's better to inform user the name change, rather than break things silently, and let user guess what happens.

Reply:
@huisedenanhai Thanks for your extensive research into this feature. You're right, I'll re-add the option.

Reply:
Done!